packages/hs-react-ui/src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,279 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import styled, { StyledComponentBase } from 'styled-components';
+import Icon from '@mdi/react';
+import { mdiCheck, mdiClose, mdiMenuDown, mdiMenuUp } from '@mdi/js';
+import { readableColor } from 'polished';
+
+import Button from '../Button';
+import { ButtonContainer } from '../Button/Button';
+import colors from '../../enums/colors';
+import timings from '../../enums/timings';
+
+const Container = styled.div<{ elevation: number }>`
+  ${({ elevation }) => {
+  const elevationYOffset = elevation && elevation >= 1 ? (elevation - 1) * 0.5 + 0.1 : 0;
+  const elevationBlur = elevation && elevation >= 1 ? (elevation / 16) * 0.1 + 0.3 : 0;
+  return `
+    width: fit-content;
+    transition: filter ${timings.slow};
+    filter: drop-shadow(0rem ${elevationYOffset}rem ${elevationBlur}rem rgba(0,0,0,${0.6 - elevation * 0.1}));
+  `;
+}}
+  
+`;
+const ValueContainer = styled(ButtonContainer)`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+  
+  line-height: initial;
```

Comment:
does it really break anything to put this change into `Button`?

Reply:
Doesn't seem like it. Generally I am trying to avoid modifying components (*goes on to modify Button), but maybe I should not have that mindset right now.

Reply:
Made the change to button and seems to not affect anything.